packages/components/psammead-section-label/src/index.jsx

```diff
@@ -57,11 +58,25 @@ const Wrapper = styled.div`
         ${({ script }) => (script ? top(script) : 'top: 0')};
       }
     `}
+
+  ${({ visuallyHidden }) =>
+    visuallyHidden &&
+    css`
+      @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+        clip-path: inset(100%);
+        clip: rect(1px, 1px, 1px, 1px);
+        height: 1px;
+        overflow: hidden;
+        position: absolute;
+        width: 1px;
+      }
+    `}
 `;
 
 Wrapper.propTypes = {
   bar: bool.isRequired,
   script: shape(scriptPropType).isRequired,
+  visuallyHidden: bool.isRequired,
```

Comment:
Curious why we think this is required, personally I'd of assumed it would be optional and default to false. Not requesting changes just a talking point 😃